### DISCUSSION TITLE
Feature request: Shortcut to read aloud after selecting text.

### DISCUSSION THREAD

**Discussion by nhan000** [travisvn/edge-tts-extension#3] [Ideas]
Instead of highlighting text and right-click and select “Read Aloud with Edge TTS”, it'd be nice if we can have set up a keyboard shortcut for that. Currently there's only a shortcut to activate the extension. Fast forward/rewind shortcuts should also be great.
![image](https://github.com/user-attachments/assets/86d135f3-ed4b-48d6-aaff-afd83d39ad7c)

**Comment by travisvn** [comment]
@nhan000 I think there are at least two separate things here

1. Adding a shortcut to initiate the read aloud function. I'll add this to my feature request log.
2. Ability to access and adjust, what would essentially be, audio track position

No1 will be much easier than no2

In any case, feature requests should really be posted in the Discussions section https://github.com/travisvn/edge-tts-chrome-extension/discussions 

I'll update the readme to reflect this better

**Comment by travisvn** [comment]
@nhan000 Aaaaaand shortcuts are here! 🙂
